templates/wizards/projects/asn1acn/CMakeLists.txt

```diff
@@ -1 +1,24 @@
-# [[ TODO: this file should be filled when working on CMake project support]]
+cmake_minimum_required(VERSION 2.8)
+
+project(%{JS: '%{ProjectName}'})
+
+set(ASN1ACNSOURCES %{JS: if ('%{AddAsnFile}' === 'true' && '%{AddAcnFile}' === 'true')
+                             '%{AsnFullFileName} %{AcnFullFileName}'
+                         else if ('%{AddAsnFile}' === 'true' )
+                             '%{AsnFullFileName}'
+                         else if ('%{AddAcnFile}' === 'true')
+                             '%{AcnFullFileName}'
+                         else
+                             ' ' })
+
+include(asn1sccSettings.txt)
+include(createFilesList.txt)
```

Comment:
After some digging through cmake docs, I came to conclusion that  `.cmake` fits here.